ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -79,6 +79,7 @@ descriptor:
   exchange: "the transfer of heat from one fluid stream to another."
   exercise: "Mode of running equipment to maintain functionality ('exercise mode')"
   exhaust: "Process or component used in the removal of air from a conditioned system to the outside atmosphere."
+  fault: "Fault of a device."
```

Comment:
Not sufficiently defined. What is a fault, and how is it specifically different from an alarm? How would someone differentiate between an alarm and a fault in practice?

Also, if faults are different than alarms but perform similar functions, these should be under point-type, not under descriptors.

Reply:
The description has been updated.

Reply:
this was removed